test/helpers/clusterlogging.go

```diff
@@ -6,7 +6,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	cl "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
-	"github.com/openshift/cluster-logging-operator/pkg/logger"
+	clolog "github.com/openshift/cluster-logging-operator/pkg/log"
```

Comment:
Why clolog? It will be much easier to search/replace if we use the same log prefix everywhere.

Reply:
It because had duplication in imports with `struct`  here: https://github.com/openshift/cluster-logging-operator/blob/master/test/helpers/types.go#L39

Reply:
OK, carry on. I think at some point we should rename the 'type log' but you don't need to do that on as part of this PR.